addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1297,10 +1312,54 @@ var SnippetsMenu = Widget.extend({
                     $snippet.addClass('o_snippet_install');
                     var $installBtn = $('<a/>', {
                         class: 'btn btn-primary o_install_btn',
-                        target: '_blank',
-                        href: '/web#id=' + moduleID + '&view_type=form&model=ir.module.module&action=base.open_module_tree',
+                        href: '#',
                         text: _t("Install"),
                     });
+                    $installBtn.on('click', function (ev) {
```

Comment:
Put the whole handler in the handlers section and a the event binding in the `events` definition at the top. No wonder you needed to use `self27` :smile:

Reply:
The :snake: will bite back.